frontend/js/models/annotation.js

```diff
@@ -46,15 +45,44 @@ define(
             defaults: function () {
                 return {
                     start: 0,
-                    duration: 0,
-                    comments: new Comments([], { annotation: this })
+
+                    // Value > 0 for time based annotation by default (contrary to point annotation)
+                    duration: 0.5,
```

Comment:
As a fix could not be introduced in time into the library, this is a workaround fixing the usability issue while still preserving the feature bern wanted (-> create/convert annotations to point annotations should remain/exist)

Point annotations can be created by setting the same start and end time (copy &paste). as this would not be user friendly, the button was introduced to handle it for the user, defaults to non-point annotation, button converts to point annotation:
![image](https://user-images.githubusercontent.com/81566097/236390493-dfc5c48d-6372-4dc3-ae40-9042329da36c.png)